gdb.commands

```diff
@@ -1,5 +1,5 @@
 target remote localhost:1234
-file dist/kernel
+file dist/debug/kernel
```

Comment:
Some or all of these can be moved to the command line, that would probably make everything easier now that I think about it

Reply:
What do you mean? I think it's fine to have it default to `dist/debug/kernel` since that's the only one with debug symbols.